src/lib/net/pinger.ts

```diff
@@ -20,7 +20,7 @@ class Pinger {
   // Resolves once a connection has been established, rejecting if
   // no connection can be made within the timeout.
   public ping = (): Promise<void> => {
-    log.debug('pinging %1:%2...', this.host_, this.port_);
+    log.debug('pinging ' + this.host_ + ':' + this.port_ + '...');
```

Comment:
For sure. That's a vestige from the s/log/console change. I thought it would be ok to leave it with the spelling above, since it's more compatible with different logger implementations (looks like console doesn't implement %n interpolation) but happy to change back.